app/scripts/controllers/transactions/index.js

```diff
@@ -701,7 +701,7 @@ export default class TransactionController extends EventEmitter {
       TOKEN_METHOD_APPROVE,
       TOKEN_METHOD_TRANSFER,
       TOKEN_METHOD_TRANSFER_FROM,
-    ].find((tokenMethodName) => tokenMethodName === name && name.toLowerCase())
+    ].find((e) => e === name && name.toLowerCase())
```

Comment:
[req] could we find a different way of fixing this line so that it doesn't lose the context of what it is we're iterating over? I mean in this case, I realize its an array of items we can directly see but I would prefer it if we didn't use single-letter variable names when possible